internal/topology/variables/schema.go

```diff
@@ -95,5 +115,31 @@ func convertToAPIExtensionsJSONSchemaProps(schema *clusterv1.JSONSchemaProps) (*
 		props.Minimum = &f
 	}
 
-	return props, nil
+	if len(schema.Properties) > 0 {
+		props.Properties = map[string]apiextensions.JSONSchemaProps{}
+		for propertyName, propertySchema := range schema.Properties {
+			p := propertySchema
+			apiExtensionsSchema, err := convertToAPIExtensionsJSONSchemaProps(&p, fldPath.Child("properties").Key(propertyName))
+			if err != nil {
+				allErrs = append(allErrs, field.Invalid(fldPath.Child("properties").Key(propertyName), "",
+					fmt.Sprintf("failed to convert schema of property: %v", err)))
```

Comment:
```suggestion
					fmt.Sprintf("failed to convert schema: %v", err)))
```
I think that's better as the path is there and "of property" sounds a bit unclear